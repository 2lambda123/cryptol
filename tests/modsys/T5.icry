:module T5::Main
:t main
main




