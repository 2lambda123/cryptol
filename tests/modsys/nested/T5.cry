module T5 where

import T4

import submodule A

a = x
