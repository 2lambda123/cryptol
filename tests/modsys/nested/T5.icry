:load T5.cry
a
:browse T4
:browse T5
