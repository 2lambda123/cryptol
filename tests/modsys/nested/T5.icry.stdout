Loading module Cryptol
Loading module Cryptol
Loading module T4
Loading module T5
0x02
Modules
=======
    
  submodule A
   
Symbols
=======
    
  x : [8]
  z : [8]
   
Symbols
=======
    
  a : [8]
   
